ibm/conns/config.go

```diff
@@ -1939,6 +1953,8 @@ func (c *Config) ClientSession() (interface{}, error) {
 		session.cisRangeAppErr = fmt.Errorf("CIS Service doesnt support private endpoints.")
 		session.cisWAFRuleErr = fmt.Errorf("CIS Service doesnt support private endpoints.")
 		session.cisFiltersErr = fmt.Errorf("CIS Service doesnt support private endpoints.")
```

Comment:
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/3544#discussion_r801290833
This PR comment is not incorporated in new PR